db/migrate/20220324220906_add_users_ref_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddUsersRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :users, null: false, foreign_key: true
```

Comment:
- [ ] Following the requirement. It will be nice to replace `:users` with `:author` as the foreign_key.